utils/domain/auctions/urls.js

```diff
@@ -0,0 +1,3 @@
+const { PREDICTION_URL, CURRENT_USER } = require('sharify').data
+
+export const liveAuctionUrl = (id) => `${PREDICTION_URL}/${id}${CURRENT_USER ? '/login' : ''}`
```

Comment:
Unfortunately `CurrentUser.orNull()` is not for use on the server and is basically calling the exact same code.
https://github.com/artsy/force/blob/master/lib/current_user.coffee#L283